code/game/objects/items/devices/scanners.dm

```diff
@@ -298,6 +300,7 @@ REAGENT SCANNER
 /obj/item/device/mass_spectrometer/adv
 	name = "advanced mass spectrometer"
 	icon_state = "adv_spectrometer"
+	item_state = "adv_spectrometer"

```

Comment:
The other option I think would be making a proper hand sprite for the basic spectrometer so there's no need for any item_state at all. I think I'll do that.